homeassistant/components/nut/config_flow.py

```diff
@@ -48,25 +44,13 @@ def _base_schema(discovery_info):
     return vol.Schema(base_schema)
 
 
-def _resource_schema_base(available_resources, selected_resources):
-    """Resource selection schema."""
+def _check_available_resouces(available_resources):
```

Comment:
I think we should remove this check. It's not the config flow responsibility to determine available sensors. The config flow should just validate the connection with the provided config data and then create the entry if the connection to the device or service is successful.

Reply:
This could eventually create integration without entity. is this OK? Before it was not possible because user was forced to select at least 1 resource to complete the config flow.

Reply:
I think we should create a device entry for the UPS explicitly if this isn't already done. Then there will always be at least a device entry. The entities is ok to not be any.

Reply:
Will you follow up with a PR to add a device explicitly to the device registry?

Reply:
Sorry, I lost this point. There is a reference to an integration that do the same, I never did it?

Reply:
Here's an example:
https://github.com/home-assistant/core/blob/c03fdd5da667e6a246555d1c7c285c60a8011d5c/homeassistant/components/bond/__init__.py#L72-L81

Reply:
Should we merge this PR, and save it for a follow up, or do you want to do it here?

Reply:
Yes, please, merge it, I will create follow up.